jet_fuel/test/models/url_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
+
+class UrlTest < ActiveSupport::TestCase
+  test "a url is valid" do
+    url = Url.create(url: "http://www.laurawhalin.com", short_url: "r832u")
+    assert url.valid?
+  end
+
+  should validate_presence_of(:url)
+  should allow_value('http://foo.com', 'http://bar.com/baz').
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.